README.md

```diff
@@ -64,7 +64,7 @@ or make sure that you add the appropriate `"devDependencies"` entries to your `p
 Then run
 
 ```sh
-tsc --init --noEmit --allowJs --allowSyntheticDefaultImports --target esnext
+tsc --init --declaration --allowJs --allowSyntheticDefaultImports --target esnext
```

Comment:
Why not just set `--declaration` and `--emitDeclarationOnly` here?

Reply:
`emitDeclarationOnly` and `noEmit` cannot be used simultaneously 

> Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.

that's why I added it directly in npm scripts 

`"type-check": "tsc --noEmit"`
 `"build": "tsc --emitDeclarationOnly ...`

Does that make sense?

Reply:
@DanielRosenwasser What do you think about my previous comment?

Reply:
Agh, okay, then this is fine, but shouldn't the command line initialization specify an `--outDir` or `--declarationDir` instead?

Reply:
Sure, I'll add `--declarationDir` in README

Reply:
@DanielRosenwasser Done, could you check, please?